teuthology/nuke/__init__.py

```diff
@@ -318,6 +318,10 @@ def nuke_helper(ctx, should_unlock):
         remote = Remote(host)
         remote.console.power_off()
         return
+    if status['machine_type'] in provision.pelagos.get_types():
```

Comment:
```suggestion
    elif status['machine_type'] in provision.pelagos.get_types():
```

Reply:
@tchaikov I wonder if this is a strict rule to use always `elif` nevertheless there is no logical error?

Reply:
i don't think this is a strict rule, i just think it's more explicit.

Reply:
fixed